core/trie/src/lib.rs

```diff
@@ -145,7 +145,7 @@ where
 /// `child_trie_root` and `child_delta_trie_root` can panic if invalid value is provided to them.
 pub fn is_child_trie_key_valid<H: Hasher>(storage_key: &[u8]) -> bool {
 	use substrate_primitives::storage::well_known_keys;
-	let has_right_prefix = storage_key.starts_with(b":child_storage:default:");
+	let has_right_prefix = storage_key.starts_with(b":child_storage:");
```

Comment:
is that true ? default is not mandatory anymore ?

Reply:
Oups, I am not sure on wether default is mandatory or not, from my perspective it seems to me that 'default' was only needed for contract.
(currently this method is use on all child trie through sr-io ChildStorageKey so I switch it to the well known key only).
May not be the proper change (if it is not I would suggest changing well_know_keys child storage prefix to use 'default').